tests/fixtures/misc/checker/logsumexp.py

```diff
@@ -0,0 +1,11 @@
+import torch
+a = torch.randn(5)
+b = torch.randn(5)
+
+# logsumexp
+y = torch.log(torch.sum(torch.exp(x), 1, keepdim=True))
+y = torch.log(torch.sum(torch.exp(2.5 + x), 1))
+
+# not logsumexp
+y = torch.log(torch.sum(torch.exp(x), 1, keepdim=True) + 2.5)
+y = torch.log(torch.sum(torch.exp(x) + 2.5, 1))
```

Comment:
Don't you want to provide a bit more anti-patterns to make sure you don't assume that `torch.log` always have a two tested functions?

```suggestion
y = torch.log(torch.sum(torch.exp(x), 1, keepdim=True) + 2.5)
y = torch.log(torch.sum(torch.exp(x) + 2.5, 1))
y = torch.log(1 + x)
```

Reply:
Added some more tests. (Not exactly `torch.log(1 + x)`, as it triggers another rule for `log1p`).